glow/src/shader/quad/core/gradient.frag

```diff
@@ -0,0 +1,69 @@
+#ifdef GL_ES
+#ifdef GL_FRAGMENT_PRECISION_HIGH
+precision highp float;
+#else
+precision mediump float;
+#endif
+#endif
+
+#ifdef HIGHER_THAN_300
+out vec4 fragColor;
+#define gl_FragColor fragColor
+#endif
+
+// #includes
```

Comment:
Since we are already hinting this here... It'd be nice to list the actual includes directly here instead of hardcoding it in Rust.